lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2832,16 +2853,31 @@ static SILValue createHostSend(SILBuilder &B, SILLocation loc, SILValue value,
   auto tensorId = createIntValue(idNumber, B, loc);
 
   auto &ctx = B.getASTContext();
-  Type scalarValueTy, tensorValueTy;
-  if (isTensorHandle(value->getType().getASTType())) {
-    tensorValueTy = value->getType().getASTType();
-    scalarValueTy = getTensorHandleElementType(tensorValueTy);
+  SubstitutionMap subMap;
+  if (isOpaqueHandle(value->getType().getASTType())) {
+    LLVM_DEBUG(llvm::dbgs() << "Sending a variant typed tensor: ");
+    LLVM_DEBUG(value->print(llvm::dbgs()));
```

Comment:
`SILValue` has `operator<<` so you shouldn't need to call `->print`. Saves two lines.

Reply:
Thanks. Done.